src/commands/games/freefire/weapon.js

```diff
@@ -0,0 +1,61 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../../')
+
+const Fuse = require('fuse.js')
+const fetch = require('node-fetch')
+
+module.exports = class FreeFireWeapon extends Command {
+  constructor (client) {
+    super({
+      name: 'weapon',
+      aliases: ['w'],
+      parent: 'freefire',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:freefire.subcommands.weapon.missingWeapon'
+      }]
+    }, client)
+  }
+
+  async run ({ t, channel, language }, query) {
+    const { locales } = await this.getData('metadata.json')
+    const locale = locales.find(locale => locale === language.slice(0, 2))
```

Comment:
This should fallback to `en` if the configured locale isn't supported by the API